app/src/main/java/com/connorlay/knitgrid/models/Stitch.java

```diff
@@ -97,4 +97,20 @@ public static void generateDefaultStitches() {
                 "come!", true).save();
         new Stitch("yo", R.drawable.yo, "Yarn over", "Instructions to come!", true).save();
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
It doesn't matter either way; if auto-generated code doesn't match the overall style, it's okay to change it. But in this case I'm indifferent.